components/input_environmental_personal.py

```diff
@@ -262,25 +262,54 @@ def input_environmental_personal(
     model_inputs = Models[selected_model].value.inputs
     model_inputs = convert_units(model_inputs, units)
 
-    values: ModelInputsInfo
-    for values in model_inputs:
-
-        if (
-            values.id == ElementsIDs.met_input.value
-            or values.id == ElementsIDs.clo_input.value
-        ):
-            inputs.append(create_autocomplete(values))
+    all_inputs = {
```

Comment:
this should not be hardcoded like this but we should get all the possible inputs from all the models.

Reply:
Yes, I am considering whether I need to modify this dictionary and store it as a class in my_config_file. It can define all the parameters that may be encountered. As long as the parameters that the model needs to be input are encountered, they will be displayed in the input form, and the rest will be hidden.
<img width="323" alt="image" src="https://github.com/user-attachments/assets/8f9ae00c-f729-47ed-b59b-49b4488e6932">
Because the parameter inputs of all models share a Callback function, although models such as Adaptive_ASHRAE do not need to transmit Clothing and Metabolic Rate in the input form (red square), they will still input the input of the drop-down options separately (green square). At this time, because each model inputs all the parameters that may be encountered, but does not display parameters other than the parameters required by the model, there will be no situation where clo_input and met_input are not defined.

Reply:
no you do not need to create another class, you should get the list of ids from the existing model class as I explain in the video